internal/util/cmdutil/cmdutil.go

```diff
@@ -113,3 +118,57 @@ func StringToImagePullPolicy(v string) fnruntime.ImagePullPolicy {
 		return fnruntime.AlwaysPull
 	}
 }
+
+// WriteFnOutput writes the output resources of function commands to provided destination
+func WriteFnOutput(dest, content string, fromStdin bool, w io.Writer) error {
```

Comment:
content can be a `io.Reader` to avoid String/byte conversion I think.

Reply:
I feel string is more intuitive and is flexible to be written using writer directly or pass it to kio filter after generating a reader.

Reply:
My observation from the code is that -- first thing we do as we enter this function is to create a new Reader from that string and a reader (bytes.Buffer) was already available at the caller of this function. This can also potentially help speed things up on packages with lot of resources.